src/shout_subgroup/modify_subgroup.py

```diff
@@ -187,3 +186,99 @@ async def subgroup_handler(update: Update, context: ContextTypes.DEFAULT_TYPE) -
     except Exception:
         logging.exception("An unexpected exception occurred")
         await update.message.reply_text("Whoops 😅, something went wrong on our side.")
+
+
+async def remove_users_from_existing_subgroup(
+        db: Session,
+        telegram_chat_id: int,
+        subgroup_name: str,
+        usernames: set[str]
+) -> SubgroupModel:
+    if not await is_group_chat(telegram_chat_id):
+        msg = f"Can't kick members from subgroup because telegram chat id {telegram_chat_id} is not a group chat."
+        logging.info(msg)
+        raise NotGroupChatError(msg)
+
+    subgroup = await find_subgroup_by_telegram_group_chat_id_and_subgroup_name(db, telegram_chat_id, subgroup_name)
+    if not subgroup:
+        msg = f"Subgroup {subgroup_name} should exist for telegram group chat id {telegram_chat_id}."
+        logging.info(msg)
+        raise SubGroupDoesNotExistsError(msg)
+
+    # If we can't find all the users, then it means we have not saved them yet.
+    users_to_be_removed = await find_users_by_usernames(db, usernames)
+    if len(users_to_be_removed) != len(usernames):
+        raise UserDoesNotExistsError("All the usernames are not in the database.")
+
+    # Find all the users who aren't in the group, then add them
```

Comment:
i think this comment wrong.
shouldn't it be to find the users in the group and remove them